src/file.rs

```diff
@@ -54,6 +55,11 @@ impl FileEntry {
         &self.name
     }
 
+    /// Returns the name of file as a byte slice, useful if the string is not utf-8.
+    pub fn name_raw(&self) -> &Vec<u8> {
```

Comment:
It would probably be more idiomatic to have this return `&[u8]`.

Reply:
i agree, thank you, will change it.